apps/video/js/video.js

```diff
@@ -785,6 +790,17 @@ function showOverlay(id) {
   });
 }
 
+function setVideoAccessibility(asButton) {
```

Comment:
Please add a comment/function description for this function as its functionality is not intuitive